docs/archived/locked-inc-ftp/figures/firmware/source/tasks/watchdog_task.c

```
@@ -379,12 +375,12 @@ watchdog_health_level_t watchdog_task_force_health_check(void)
 }
 
 esp_err_t watchdog_task_register_task(const TaskHandle_t task_handle,
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'watchdog_task_register_task' of similar type ('const int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)